src/towncrier/newsfragments/640.misc

```diff
@@ -0,0 +1,4 @@
+Moved towncrier version definition from src/towncrier/_version.py to pyproject.toml
+
+towncrier.__version__ was removed, after being deprecated in 23.6.0.
```

Comment:
these 2 lines needs to be in a separate newsfragment file of type `.removal`

Reply:
done